pkg/github/github_mock.go

```diff
@@ -255,7 +255,7 @@ func (m *MockGitHubClient) RequestReviewers(ctx context.Context, owner, repo str
 	}
 
 	return &RequestReviewersResponse{
-		Users: m.UserReviewers,
-		Teams: m.TeamReviewers,
+		Users: append(m.UserReviewers, users...),
+		Teams: append(m.TeamReviewers, teams...),
```

Comment:
I think you need to lock these before updating them. Regardless, I dont think we want to append the users/teams, either have a static response (m.UserReviewers) or just return the inputs as we dont really care what this is doing, we just want to expect a specific response.